src/Executables/ExportCoordinates/ExportCoordinates.hpp

```diff
@@ -80,13 +80,19 @@ struct ExportCoordinates {
     // Collect volume data
     // Remove tensor types, only storing individual components
     std::vector<TensorComponent> components;
-    components.reserve(Dim);
+    components.reserve(Dim + 1);
     for (size_t d = 0; d < Dim; d++) {
       components.emplace_back(element_name + "InertialCoordinates_" +
                                   inertial_coordinates.component_name(
                                       inertial_coordinates.get_tensor_index(d)),
                               inertial_coordinates.get(d));
     }
+    // Also output the determinant of the inverse jacobian, which measures
+    // the expansion and compression of the grid
+    const auto& det_inv_jac =
+        db::get<domain::Tags::DetInvJacobian<Frame::Logical, Frame::Inertial>>(
+            box);
+    components.emplace_back(element_name + "DetInvJacobian", get(det_inv_jac));
```

Comment:
It's probably better to replace `"DetInvJacobian"` -> `db::tag_name<domain::Tags::DetInvJacobian<Frame::Logical, Frame::Inertial>>()` (this is how we label fields in `ObserveFields`, for instance)